### PR TITLE
Fix #702 Update key-vis.md

### LINKED ISSUE #702: 第三部分 2.1 识别集群热点和业务模式
请按照 [审核标准](https://github.com/pingcap-incubator/tidb-in-action/blob/master/README.md) 审核以下章节：
第三部分 [2.1 识别集群热点和业务模式](https://github.com/pingcap-incubator/tidb-in-action/tree/master/session3/chapter2/key-vis.md)

### PR DESCRIPTION
**Pull request by wulinning133**
### What problem does this PR solve? 
This PR is try to fix [#702](https://github.com/pingcap-incubator/tidb-in-action/issues/702)

### How to contact you? 
- Email:
- Telephone:
- WeChat:wulinning
- Others:

### CODE REVIEW

File: session3/chapter2/key-vis.md [hunk 1]
```diff
@@ -52,8 +50,8 @@ KeyVis 提供了一个简单，直观，符合直觉的方式来观测整体 TiD
 在这个时间段上读写流量较低，颜色越亮（hot）表示读写流量越高，即越热。如果热图中的
 颜色太亮或太暗，则可能很难观察到细节。此时，我们可以点击 Brightness 按钮，然后通过
 滑块来调节页面的亮度。
-注：显示一个区域内的热图时，会根据区域内的流量情况来界定冷热。当整个区域流量较为平
-均时，即使整体流量在数值上很低，您依然有可能观察到较大的亮色区域。请注意一定要结合
+注：显示一个区域内的热图时，会根据区域内的流量情况来界定冷热。当整个区域流量不够
+均匀时，即使整体流量在数值上很低，您依然有可能观察到较大的亮色区域。请注意一定要结合
```
Comment (AndreMouche) [inline session3/chapter2/key-vis.md:54]: 第 L59 行
当我们需要实时观察数据库的流量分布情况是，=》当我们需要实时观察数据库的流量分布情况时

Comment (AndreMouche) [reply session3/chapter2/key-vis.md:54]: 第 L60 行，自动刷新会被关闭

**Review by AndreMouche — COMMENTED** [review]
Rest LGTM

**Comment by wulinning133** [comment]
重新修订了下

**Review by ngaut — APPROVED** [review]
LGTM

**Pull request merged**